README.md

```diff
@@ -1,4 +1,61 @@
 # meta
-Meta issues concerning many/all the Plone repositories.
 
-See the [Issues](https://github.com/plone/meta/issues) for details.
+This repository's aim is to define common configuration files
+across all/most repositories in the [GitHub plone organization](https://github.com/plone).
+
+This allows Plone developers and contributors
+to have the same expectations/workflow
+when contributing to any of the Plone repositories.
+
+The following areas are configured on repositories
+where the configuration has been applied.
+
+## Testing
+
+A `tox.ini` configuration is added to each repository to allow contributors
+to test their contributions, as well as running the tests on GitHub Actions.
+
+To run the tests, run the following commands:
+
+```shell
+python3.11 -m venv venv
+. venv/bin/activate
+pip install tox
+tox -e test
+```
+
+Note: the GitHub Actions workflow is __not__ installed on each repository,
+but rather shared by all repositories.
+
+The workflow definition can be found in this repository in `shared-workflows/tests.yml`.
```

Comment:
Can you link to the organisation settings page where we need to enable the workflow for a repository?
I always have trouble finding it.

Reply:
Good idea 👍🏾